tests/robustness/checker/checker.go

```diff
@@ -84,29 +90,25 @@ type SnapshotMetadata struct {
 
 // GetSnapshotMetadata gets the metadata associated with the given snapshot ID.
 func (chk *Checker) GetSnapshotMetadata(snapID string) (*SnapshotMetadata, error) {
+	chk.mu.RLock()
+	defer chk.mu.RUnlock()
+
 	return chk.loadSnapshotMetadata(snapID)
 }
```

Comment:
Could you either (i) eat up deleted snapshots or (ii) provide an IsDeleted method on `SnapshotMetadata`?

Reply:
[Added here](https://github.com/kastenhq/kopia/pull/125/commits/65c1759c106d5b9fa826198c5b1dbb64ec098d88)